frontend/src/components/Logs/RawLogView/styles.ts

```diff
@@ -35,19 +35,23 @@ interface RawLogContentProps {
 	linesPerRow: number;
 	$isReadOnly: boolean;
 	$isActiveLog: boolean;
+	$isTextOverflowEllipsisDisabled: boolean;
 }
 
 export const RawLogContent = styled.div<RawLogContentProps>`
 	margin-bottom: 0;
 	font-family: Fira Code, monospace;
 	font-weight: 300;
 
-	overflow: hidden;
-	text-overflow: ellipsis;
-	display: -webkit-box;
-	-webkit-line-clamp: ${(props): number => props.linesPerRow};
-	line-clamp: ${(props): number => props.linesPerRow};
-	-webkit-box-orient: vertical;
+	${(props): string =>
```

Comment:
Destruction props at parameter level. 
```suggestion
	${({ isTextOverflowEllipsisDisabled,  linesPerRow }): string =>
```